src/components/externalLinks/ExternalLinksCRUD/AddExternalLink.scss

```diff
@@ -76,43 +98,28 @@
                         }
                     }
 
-                    .link-monitoring-tool {
-                        display: flex;
-                        align-items: flex-start;
+                    .configure-link-action-content {
+                        display: grid;
+                        row-gap: 8px;
 
                         .link-name {
                             display: flex;
                             flex-direction: column;
-                            width: 200px;
 
                             input {
-                                width: 200px;
-                                height: 32px;
+                                height: 36px;
                                 padding: 10px 6px;
                                 background-color: var(--N50);
                                 color: var(--N900);
                                 border: 1px solid var(--N200);
                                 border-radius: 4px;
                             }
                         }
+                    }
 
-                        .monitoring-tool {
-                            width: 150px;
-
-                            .absolute-option {
-                                position: absolute;
-                                padding: 0 10px;
-                            }
-                        }
-
-                        .link-clusters {
-                            width: 278px;
-                        }
-
-                        .link-delete {
-                            width: 20px;
-                            height: 20px;
-                        }
+                    .link-delete {
+                        width: 20px;
```

Comment:
Here also cant we use `icon-dim-20` instead

Reply:
Done